### PR TITLE
Python3: fix several imports, and run 2to3 on s3cmd

### PR DESCRIPTION
**Pull request by shaform**
Continue the effort for making s3cmd Python 3 compatible.

- Remove remaining implicit relative imports
I thought about removing the circular dependencies, but in the end I decided to simply modify the imports to minimize changes.
- Fix some imports that have different names in Python 3.
- Run 2to3 on s3cmd to fix several compatibility issues

### CODE REVIEW

File: s3cmd [hunk 1]
```diff
@@ -2110,9 +2116,9 @@ def cmd_fixbucket(args):
                 debug("Step 3:  ... then to %s" % key_new)
                 src = S3Uri("s3://%s/%s" % (culprit.bucket(), key_bin))
                 dst = S3Uri("s3://%s/%s" % (culprit.bucket(), key_new))
-		if cfg.dry_run:
-		    output("[--dry-run] File %r would be renamed to %s" % (key_bin, key_new))
-		    continue
```
Comment (shaform) [inline s3cmd]: Notice that here are some strange TABs.

**Comment by fviard** [comment]
Thanks.
Before merging, there is just a point that annoys me, but I don't know if it results of the usage of 2to3 or if it is needed by python 3 for a reason. Please see my questions inline.

File: S3/Utils.py [hunk 2]
```diff
@@ -33,8 +35,8 @@
     sys.stderr.flush()
     sys.exit(EX_OSFILE)
 
-import Config
-import Exceptions
+import S3.Config
```
Comment (shaform) [inline S3/Utils.py]: These are needed because of circular import dependency. If not for this reason `from . import Config` is certainly better.

Comment (shaform) [inline S3/Utils.py]: In the long run, it might be better to remove all those circular import dependencies. But doing so might change some of the APIs.

**Comment by fviard** [comment]
Thank you!

**Pull request merged**